### DISCUSSION TITLE
Is there a way to access all websites?

### DISCUSSION THREAD

**Discussion by plumkewe** [wibyweb/wiby#9] [Q&A] [answered]
I'm a student who is quite enthusiastic about WEB 1.0, and I want to conduct my research on the changes in the WEB. Since I haven't yet worked with databases, my question is: is there a simple way to access all websites in the database?

**Comment by wibyweb** [comment, marked as answer]
Hello. 

A list of every indexed website is not publicly available. The two ways to access the database are through the search form or the JSON output located at /json/. If you want more historic archived pages to see, I would recommend also checking out theoldnet.

**Comment by CouldBeThis** [comment]
@plumkewe have you checked out archive.org's [wayback machine](https://web.archive.org/)? They have an [API](https://archive.org/help/wayback_api.php). And you can download archived websites as WARC files which are browseable offline with a WARC viewer. 

Checking in on the Wayback Machine's updates, I even find this article: [From Fake News to Open Data: Studying the Histories of Digital Media Using the Wayback Machine | Internet Archive Blogs](https://blog.archive.org/2023/10/09/from-fake-news-to-open-data-studying-the-histories-of-digital-media-using-the-wayback-machine/) which sounds similar to what you are asking about. maybe it would be helpful to have a look at their methods.
